internal/client.go

```diff
@@ -436,6 +431,13 @@ type (
 		// Optional parameter that is designed to be used *in tests*. It gets invoked last in
 		// the gRPC interceptor chain and can be used to induce artificial failures in test scenarios.
 		TrafficController TrafficController
+
+		// Interceptors to apply to some calls of the client. Any interceptors that
+		// also implement Interceptor (meaning they implement WorkerInterceptor in
+		// addition to ClientInterceptor) will be used for worker interception as
+		// well. When worker interceptors are here and in worker options, the ones
+		// here wrap the ones in worker options.
+		Interceptors []ClientInterceptor
```

Comment:
I should make it clear earlier interceptors wrap later ones in this slice (same for worker options).

Reply:
Maybe just add a comment warning not to register the same interceptors twice, here and in the worker.

Reply:
Will also do that

Reply:
Clarified